Lib/src/main/java/uk/ac/cam/cl/wildpetscience/triton/lib/config/ConfigServer.java

```diff
@@ -63,11 +64,22 @@ public static void start(int port, Map<String, HTTPRoute> routes) {
      */
     public static void start(int port) {
         Map<String, HTTPRoute> map = new HashMap<>();
+
+        Spark.staticFileLocation("/serve");
+
         map.put("/", new HTTPRoute(new RootConfigRoute(), HTTPMethod.GET));
-        map.put("/:resource", new HTTPRoute(new StaticResourceRoute()));
-        map.put("/:resource/*", new HTTPRoute(new StaticResourceRoute(), HTTPMethod.GET));
+
         map.put("/start", new HTTPRoute(new StartStopRoute(), HTTPMethod.POST));
         map.put("/stop", new HTTPRoute(new StartStopRoute(), HTTPMethod.POST));
+
+        try {
+            ImageInputSource in = new WebcamInputSource();
```

Comment:
This won't work on the Pi - this is fine for now but we must remember to pass in the correct implementation when writing the Pi specific stuff. Perhaps the start method can take a Driver (since we want to have one driver for the whole thing).